src/content/1.7/scale/_index.md

```diff
@@ -7,6 +7,34 @@ chapter: true
 
 ### Chapter 8
 
-# How to make PrestaShop scale
+# Scale
 
-Follow this guide to optimize your PrestaShop shop.
+Scaling your PrestaShop instalation is something we hope every merchant will need, because it means her business is growing.
+
+Hopefully, there are plenty advices in these pages that will help.
+
+# Scalings
+
+Before doing anything to your PrestaShop setup, there are several things you need to know.
+
+First, to be aware there are several ways to scale any application, PrestaShop incuded.
+
+## Vertical scaling
+
+The most used one is called vertical scaling, where you boost your server's performance : you add some RAM, some CPU, increase the disks performances, and so on. It can be quick and easy if your hoster allows it and does not require any change on your application setup and configuration.
+
+Though, it's not the more efficient one and doubling the server's resources will harshly double its performances.
+
+Also, pricing being what it is, vertical scaling may not be linear (depending on your hosting company, your mileage may vary).
+
+Doubling your server capacity _may_ cost you more than twice its original price.
+
+## Horizontal scaling
+
+The most efficient way to scale an application remains adding more servers.
+
+It may not be as simple as it sounds as most PrestaShop file require being shared (or at least synchronized) across the instances, but it's not that hard when you have figured it out.
```

Comment:
```suggestion
It may not be as simple as it sounds, as PrestaShop requires many files being shared (or at least synchronized) across instances, but it's not that hard when you have figured it out.
```